### DISCUSSION TITLE
Assimp include directory incorrect - can't include files within include/assimp

### DISCUSSION THREAD

**Discussion by jasonrdunne** [assimp/assimp#4894] [Q&A] [answered]
I've downloaded and built multiple versions of assimp now (5.2.5 and 4.1.0) with cmake. No matter what, every import statement within include/assimp, such as #include <types.h>, gives the error "Cannot open include file: 'assimp/types.h'." 
If i manually replace the include with the absolute path, it works.
Wtf is this. How do i make the include folder include itself?

**Comment by kimkulling** [comment, marked as answer]
Have you added your include folders correctly? And you need to use

`#include <assimp/types.h`

when you just have added the include folder to your include var.
